freeipa/src/main/resources/schema/app/20230227094552_CB-20943_Remove_the_secrets_from_the_userdata_script_which_are_stored_in_the_database.sql

```diff
@@ -0,0 +1,20 @@
+-- // CB-20943 Remove the secrets from the userdata script which are stored in the database
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE image ADD COLUMN IF NOT EXISTS gatewayuserdata TEXT;
+ALTER TABLE image ADD COLUMN accountid VARCHAR(255);
+
+ALTER TABLE image_history ADD COLUMN gatewayuserdata TEXT;
+ALTER TABLE image_history ADD COLUMN accountid VARCHAR(255);
+
+UPDATE image i SET accountid = s.accountid FROM stack s WHERE s.id = i.stack_id;
```

Comment:
you could filter for stacks which are not deleted to make the update a bit smaller

Reply:
id the cluster deleted then image also deleted

Reply:
I don't see any imageentity deletion, and we don't delete the stack from db, just set some flags. but correct me if I'm missing something